src/services/cronJobs/checkProjectVerificationStatus.ts

```diff
@@ -0,0 +1,183 @@
+    project.verified = false;
+  } else if (
+    // Projects that already expired verification are given a last chance
+    // for this feature
+    project.projectUpdate.createdAt <= maxDaysForSendingUpdateLastWarning &&
+    project.verificationStatus === null
+  ) {
+    project.verificationStatus = RevokeSteps.UpForRevoking;
+  } else if (
+    // Projects that had the last chance and failed to add an update are revoked
+    project.projectUpdate.createdAt <= maxDaysForRevokingBadge &&
+    project.verificationStatus === RevokeSteps.LastChance
+  ) {
+    project.verificationStatus = RevokeSteps.Revoked;
+    project.verified = false;
+  } else if (
+    // projects that were warned are sent a last chance warning
+    project.projectUpdate.createdAt <= maxDaysForSendingUpdateLastWarning &&
+    project.projectUpdate.createdAt > maxDaysForRevokingBadge &&
+    project.verificationStatus === RevokeSteps.Warning
+  ) {
+    project.verificationStatus = RevokeSteps.LastChance;
+  } else if (
+    // After reminder at 60/75 days
+    project.projectUpdate.createdAt <= maxDaysForSendingUpdateWarning &&
+    project.projectUpdate.createdAt > maxDaysForSendingUpdateLastWarning
+  ) {
+    project.verificationStatus = RevokeSteps.Warning;
+  } else if (
+    // First email for reminding to add an update
+    project.projectUpdate.createdAt <= maxDaysForSendingUpdateReminder &&
+    project.projectUpdate.createdAt > maxDaysForSendingUpdateWarning
+  ) {
+    project.verificationStatus = RevokeSteps.Reminder;
+  }
+
+  await project.save();
+
+  // save status changes history
+  if (project.verificationStatus === RevokeSteps.Revoked) {
+    await Project.addProjectStatusHistoryRecord({
+      project,
+      status: project.status,
+      description: HISTORY_DESCRIPTIONS.CHANGED_TO_UNVERIFIED_BY_CRONJOB,
+    });
+  }
+
+  const user = await User.findOne({ id: Number(project.admin) });
+
+  // segment notifications
+  const segmentProject = {
+    email: user?.email,
+    title: project.title,
+    lastName: user?.lastName,
+    firstName: user?.firstName,
+    OwnerId: user?.id,
+    slug: project.slug,
+    walletAddress: project.walletAddress,
+    description: project.description,
+  };
+
+  analytics.track(
+    selectSegmentEvent(project.verificationStatus),
+    `givethId-${user?.id}`,
+    segmentProject,
+    null,
+  );
+};
+
+const selectSegmentEvent = projectVerificationStatus => {
+  switch (projectVerificationStatus) {
+    case RevokeSteps.Reminder:
+      return NOTIFICATIONS_EVENT_NAMES.PROJECT_BADGE_REVOKE_REMINDER;
+    case RevokeSteps.Warning:
+      return NOTIFICATIONS_EVENT_NAMES.PROJECT_BADGE_REVOKE_WARNING;
+    case RevokeSteps.LastChance:
+      return NOTIFICATIONS_EVENT_NAMES.PROJECT_BADGE_REVOKE_LAST_WARNING;
+    case RevokeSteps.Revoked:
+      return NOTIFICATIONS_EVENT_NAMES.PROJECT_BADGE_REVOKED;
+    case RevokeSteps.UpForRevoking:
+      return NOTIFICATIONS_EVENT_NAMES.PROJECT_BADGE_UP_FOR_REVOKING;
```

Comment:
@CarlosQ96  @WhyldWanderer @divine-comedian 
have we created the autopilot journeys for these events?